File: Aoc.Solutions/Solutions/Day00.cs
using System.Collections.Generic;
using System.Linq;

using Aoc.Runner;

namespace Aoc.Solutions {
    public class Day00 : Day {
        public override string SolveA(string input) {
            return "SolveA";
        }

        public override string SolveB(string input) {
            return "SolveB";
        }
        public Day00() {
            Tests = new()
            {
                new("1", "input", "SolveA", (input) => SolveA(input))
            };

        }
    }
}

File: Aoc.FsSolutions/Solutions/Day00.fs
module Aoc.Solutions.Day00

open Aoc.Runner
open System.Collections.Generic
open System.Collections

type Day00() =
    inherit Day()

    override this.SolveA(input) = "A"

    override this.SolveB(input) = "B"

    override this.Tests =
        [ Test("fsharp test", "123", "123", (fun input -> "123")) ]
        |> List<Test>
